db/schema.sql

```diff
@@ -0,0 +1,30 @@
+/*  Execute this file from the command line by typing:
+ *    mysql -u root < schema.sql
+ *  to execute the queries in this file.
+ */
+
+CREATE DATABASE IF NOT EXISTS booking_checkout;
+
+USE booking_checkout;
+
+CREATE TABLE rooms (
+  room_id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  nightly_price INT NOT NULL,
+  past_week_views INT NOT NULL,
+  cleaning_fee INT NOT NULL,
+  service_fee INT NOT NULL,
+  last_updated DATE,
+  review_count INT NOT NULL,
+  avg_rating INT NOT NULL
+);
+
+CREATE TABLE bookings (
```

Comment:
how are you dealing with blackout dates per toby's comments?

Reply:
I added a column "blackout" to the bookings table. It takes a boolean value (0 for false and 1 for true). If a row has a blackout value of "true", it means the room is unavailable, but I will still be able to set that row's dates in the bookings table to indicate whether the room is unavailable even though it is not booked.